konan_sdk/endpoints/base_endpoint.py

```diff
@@ -96,3 +119,35 @@ def get(self, api_url: str):
         self.process_response()
 
         return self.response
+
+
+class KonanBaseAuthenticatedEndpoint(KonanBaseEndpoint):
+    def __init__(self, api_url: str, user: KonanUser = None, **kwargs) -> None:
+        super().__init__(api_url, user=user, **kwargs)
```

Comment:
`user=user` might be unnecessary here as `KonanBaseEndpoint` doesn't use it from kwargs or expects it

Reply:
True .. fixed it